content/articles/creating-an-application-using-beeware/index.md

```diff
@@ -0,0 +1,259 @@
+    └── simplecalculator.dist-info
+        ├── INSTALLER
+        └── METADATA
+
+
+```
+The `src` folder has files for running the application. All the logic for running the application are found in the `app.py`. The `app.py` should look as follows:
+
+`app.py`
+```python
+import toga
+from toga.style import Pack
+from toga.style.pack import COLUMN, ROW
+
+class SimpleCalculator(toga.App):
+    def startup(self):
+        main_box = toga.Box()
+        self.main_window = toga.MainWindow(title=self.formal_name)
+        self.main_window.content = main_box
+        self.main_window.show()
+    
+def main():
+    return SimpleCalculator()
+```
+The file begins by importing `toga` toolkit. Then we define a class `SimpleCalculator` that conatins a `startup` method. This method defines a toga box component.It serves as the main box.The name `main_box` is declared and initialized by default when you create the application but may be changed later 
+as desired.
+
+Then we declare a main window whose title is the app name we defined while creating after running the command `briefcase new`. We then have the window contain our empty main box as it's content. Then we have our application show our window. We finally define a `main` function that returns our `SimpleCalculator` class instance. This `main` method is called by the `__main__.py` file and invoked by it.
+
+By now you have a simple working application.You can cd into `Simple Calculator` and type `briefcase dev` to run the app in developer mode. You should have the following simple application:
+
+![Starter app image](/engineering-education/content/articles/creating-an-application-using-beeware/starter.png)
+
+
+### Setting up the boxes
+We will now modify the `app.py` file step by step in order to create our final calculator application.We will start by defining all the necessary box components required. Modify the `app.py` file as follows:
+
+```python
+import toga
+from toga.style import Pack
+from toga.style.pack import COLUMN, ROW
+from functools import partial
+
+class SimpleCalculator(toga.App):
+
+    def startup(self):
+        box1 = toga.Box()
+        box2 = toga.Box()
+        box3 = toga.Box()
+        box4 = toga.Box()
+        box5 = toga.Box()
+        box6 = toga.Box()
+
+        main = toga.Box()
+
+        # adding in main box
+        main.add(box1)
+        main.add(box2)
+        main.add(box3)
+        main.add(box4)
+        main.add(box5)
+        main.add(box6)
+        main.style.update(direction=COLUMN)
+
+        self.main_window = toga.MainWindow(title=self.formal_name)
+        self.main_window.content = main
+        self.main_window.show()
+
+
+def main():
+    return SimpleCalculator()
+
+```
+We begin by creating six boxes and a main box that will have all the six boxes inside it. We  have the main box as a column box, meaning it will have all width by default unless defined and height will expand according to the content within the box.
```

Comment:
```suggestion
We begin by creating six boxes and one main box that will have all the six boxes inside. We have the main box as a column box, meaning it will have all width by default unless defined and height will expand according to the content within the box.
```